lib/hammer_cli_foreman_openscap/arf_report.rb

```diff
@@ -6,8 +6,16 @@ class ArfReport < HammerCLIForeman::Command
     class ListCommand < HammerCLIForeman::ListCommand
       output do
         field :id, _("Id")
-        field :host, _("Host")
         field :reported_at, _("Reported at")
+        from :host do
+          field :name, _("Host name")
+        end
+        from :openscap_proxy do
+          field :name, _("Openscap proxy name"), :hide_blank => true
```

Comment:
it does not seem to work, I'm getting `Error: undefined method `new' for {:hide_blank=>true}:Hash`, we can probably live with empty fields for now so feel free to remove it (sorry for back and forth). good to merge afterwards